libs/csm/rest/csm_rest_quota.py

```diff
@@ -207,3 +207,11 @@ def get_user_capacity_usage(self, uid, resource,
                                           headers=header)
         self.log.info("Get user quota request successfully sent...")
         return response
+
+    def get_iam_user_payload(self):
```

Comment:
move this to some Lib or make add @staticmethod and remove (self)

Reply:
Done!